augraphy/augmentations/inkshifter.py

```diff
@@ -95,7 +95,36 @@ def noise_map(self, shape, res=(64, 64)):
         d0, d1 = shape[0] // res[0], shape[1] // res[1]
         angles = 2 * np.pi * np.random.rand(res[0] + 1, res[1] + 1)
         grad = np.dstack((np.cos(angles), np.sin(angles)))
+        gysize, gxsize = grad.shape[:2]
         grid = np.mgrid[: res[0] : res[0] / shape[0], : res[1] : res[1] / shape[1]].transpose(1, 2, 0) % 1
```

Comment:
@ss756 I found an issue here where `grid` size doesn't match with transformed `grad` size, and this is mostly due to the `np.ceil` in line 93 above. I added the evaluations below to prune or pad their size so that their size are always consistent. If you have a better solution, please push the update too.